manifest/io.go

```diff
@@ -16,7 +24,28 @@ func (m *Manifest) ToJson() ([]byte, error) {
 
 func ManifestFromYaml(content []byte) (manifest Manifest, err error) {
 	err = yaml.Unmarshal(content, &manifest)
-	return
+	if err != nil {
+		return manifest, err
+	}
+
+	return manifest, manifest.Validate()
+}
+
+func (m Manifest) Validate() error {
+	for i, mount := range m.Mounts {
+		if mount.BaseDir != "" {
+			if !filepath.IsAbs(mount.BaseDir) {
+				return fmt.Errorf("BaseDir needs to be absolute path")
+			}
+			if !strings.HasSuffix(mount.BaseDir, "/") {
+				mount.BaseDir = mount.BaseDir + "/"
+			}
```

Comment:
I'd say that is not strictly required as there is `filepath.Join()` which can be used